lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -219,4 +228,19 @@ defmodule LambdaEthereumConsensus.ForkChoice do
       Logger.debug("[Fork choice] Store persisted")
     end)
   end
+
+  defp persist_fork_choice_store(store) do
+    :telemetry.span([:fork_choice, :persist], %{}, fn ->
```

Comment:
Let's have the telemetry call on the db methods directly, so they are called no matter who is storing/fetching the store.

Reply:
Done!